change/@fluentui-react-charting-3dfa3e61-d8ae-4541-a3ed-1bb4ae87e2c1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30509&commitId=55b8832a2e3d9843f098ce218b789ffd0dadc625&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-SankeyChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.Basic - Dark Mode.default.chromium.png|263720| Changed|
|react-charting-SankeyChart.Basic - RTL.default.chromium.png|502049| Changed|
|react-charting-SankeyChart.PlaceHolder.default.chromium.png|467554| Changed|
|react-charting-SankeyChart.PlaceHolder - RTL.default.chromium.png|498821| Changed|
|react-charting-SankeyChart.Basic.default.chromium.png|471553| Changed|
|react-charting-SankeyChart.PlaceHolder - Dark Mode.default.chromium.png|267931| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>